components/table-row/index.js

```diff
@@ -1,14 +1,37 @@
-import React, {useCallback} from 'react'
+import React, {useCallback, useEffect, useState} from 'react'
 import PropTypes from 'prop-types'
-import {Table, Checkbox} from 'evergreen-ui'
+import {Table, Checkbox, Pane, Spinner, Tooltip, WarningSignIcon} from 'evergreen-ui'
+
+import {getNumeros} from '@/lib/bal-api'
 
 import TableRowActions from '@/components/table-row/table-row-actions'
 import TableRowEditShortcut from '@/components/table-row/table-row-edit-shortcut'
 import TableRowNotifications from '@/components/table-row/table-row-notifications'
 
-const TableRow = React.memo(({label, nomAlt, complement, secondary, notifications, isSelected, isEditingEnabled, handleSelect, actions}) => {
+const TableRow = React.memo(({label, nomAlt, complement, voieId, secondary, notifications, isSelected, isEditingEnabled, handleSelect, actions}) => {
   const {onSelect, onEdit} = actions
 
+  const [hasNumeros, setHasNumeros] = useState()
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
```

Comment:
Afin d’éviter de voir l’icône apparaître sur toutes les voies à chaque chargement, je propose de transformer la variable `hasNumeros` en `hasNoNumero` , ce qui permet d’avoir l’icône masquée par défaut : 

```suggestion
  useEffect(() => {
    const fetchNumeros = async () => {
      try {
        if (voieId) {
          const numeros = await getNumeros(voieId)
          setHasNoNumeros(numeros.length === 0)
        } else {
          setHasNoNumeros(false)
        }
      } catch (error) {
        console.log(error)
      }
    }

    fetchNumeros()
    setIsLoading(false)
  }, [voieId])
```